### ISSUE TITLE
Chart image download doesn't work in high contrast mode

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#710]
When chart image is downloaded in high contrast mode, hardly anything shows on downloaded image:
![image](https://user-images.githubusercontent.com/43404358/82557600-ebe5e080-9b63-11ea-915e-f94012380c72.png)

**Comment by brockfanning** [comment]
Ah, good catch.

Tech note: In our `_download_buttons.scss` file, there is a class called `image-download-in-progress`. Anything in there should apply to the downloaded image. I suspect we'll need to make the text black in there, overriding the normal white color of high-contrast mode.

**Comment by AnnCorp** [comment]
@brockfanning @LucyGwilliamAdmin is this still a bug? If so think could add to 1.3
